src/components/app/data/hooks/useBFF.test.jsx

```diff
@@ -129,90 +116,52 @@ const mockBFFDashboardData = {
 };
 
 describe('useBFF', () => {
-  const Wrapper = ({ children }) => (
+  const Wrapper = ({ initialEntries = [], children }) => (
     <QueryClientProvider client={queryClient()}>
-      {children}
+      <MemoryRouter initialEntries={initialEntries}>
+        <Routes>
+          <Route path=":enterpriseSlug" element={children} />
+          <Route path=":enterpriseSlug/search" element={children} />
```

Comment:
nit: is there a need to render different contents depending on the page route rendered? E.g., if we test assertions on content rendered within `children`, how will the test know that content was rendered on the dashboard vs. search route?

Likely similar question for other tests' route definitions.